lib/src/simulations/simulations.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import '../custom_items/simulation_card.dart';
+import 'toothpick.dart';
+import 'langton_ant.dart';
+
+class Simulations {
+  static final allSimulations = <Widget>[
+    SimulationCard(
+      simulationName: 'ToothPick Pattern',
+      image: 'images/ToothpickPattern.gif',
+      direct: ToothpickPattern(),
+      infoLink: 'https://en.wikipedia.org/wiki/Toothpick_sequence',
+    ),
+    SimulationCard(
+      simulationName: 'Langton\'s Ant',
+      image: 'images/LangtonsAnt.gif',
+      direct: LangtonAnt(),
+      infoLink: 'https://en.wikipedia.org/wiki/Langton%27s_ant',
+    ),
+  ];
+
+  static List<Widget> returnWidgets(List<int> a) {
```

Comment:
Instead of sending a single list as an argument, we can send two parameters, one being an `int`, the other being a `list` with default value `NULL`. When only an int is passed, widgets having ID up to that number would be shown. If both are passed, widgets up to the `int` value, but excluding the ones present in the `list` would be shown.

I guess that will look better? As of now, we have only 2 widgets, and so the passed argument is small. In case of a large number of widgets, the passed list will look not-so-neat and occupy more space.

Sounds good?